### DISCUSSION TITLE
I need help to launch ssh brute force attacks on TPOT

### DISCUSSION THREAD

**Discussion by ghost** [telekom-security/tpotce#894] [Q&A]
I have successfully set up TPot on a Google cloud instance. It's working and displaying captured attacks. However, I need to make a demonstration for a school project and my ssh attacks are not being logged in kibana. Don't get me wrong, it's working, but it's not logging mine. I don't know what I'm doing wrong...

**Comment by zhovner** [comment]
Make sure you are attacking port 22. You can read the full log here:

`/data/cowrie/log/cowrie.json`

**Comment by t3chn0m4g3** [comment]
Please review the closed issues. GCP needs some tweaking to work properly. I will convert this issue to a discussion in the hopes someone can help.
